frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPaletteContent/CommandPaletteContent.tsx

```diff
@@ -72,27 +77,29 @@ export const CommandPaletteContent: FC<Props> = ({ closeDialog }) => {
       <div className={styles.main}>
         <Command.List>
           <Command.Empty>No results found.</Command.Empty>
-          <Command.Group heading="Tables">
-            {Object.values(schema.current.tables).map((table) => (
-              <Command.Item key={table.name} value={table.name} asChild>
-                <a
-                  href={getTableLinkHref(table.name)}
-                  onClick={(event) => {
-                    // Do not call preventDefault to allow the default link behavior when ⌘ key is pressed
-                    if (event.ctrlKey || event.metaKey) {
-                      return
-                    }
+          {inputMode.type === 'default' && (
```

Comment:
We are plan to show/hide option with input mode as like this. The table options will be displayed with `inputMode.type === "default"`, and the command options will be with `inputMode.type === "default" || inputMode.type === ''command'`.

This change won't affect the production since `inputMode.type` will be "default" permanently with the current implementation.